pkg/compiler/import_test.go

```diff
@@ -61,3 +65,27 @@ func TestImportNameSameAsOwn(t *testing.T) {
 	}`
 	eval(t, src, big.NewInt(3))
 }
+
+func TestImportCycleIndirect(t *testing.T) {
```

Comment:
This one should be probably called `TestImportCycleDirect` (pkg2 imports pkg3 and pkg 3 imports pkg2).

Reply:
Ah, yeah, fixed.